src/command/status.ts

```diff
@@ -61,6 +70,28 @@ export class Status extends RootCommand implements LeafCommand {
     }
   }
 
+  private async checkTopology(): Promise<{
+    connected: number
+    population: number
+    depth: number
+  }> {
+    try {
+      const { connected, population, depth } = await this.beeDebug.getTopology()
+
+      return {
+        connected,
+        population,
+        depth,
+      }
+    } catch {
+      return {
+        connected: 0,
```

Comment:
This seems to return zeroes if it cannot connect to the debug API. However it's possible that there are many connections but the debug API is disabled. 

I think it would be better to not show the topology information when there is a problem with the debug API instead of showing possibly incorrect information.